libs/accounts/src/lib/accounts-table.tsx

```diff
@@ -0,0 +1,146 @@
+import { forwardRef } from 'react';
+import type { ColumnApi, ValueFormatterParams } from 'ag-grid-community';
+import {
+  PriceCell,
+  formatNumber,
+  t,
+  addSummaryRows,
+} from '@vegaprotocol/react-helpers';
+import { AgGridDynamic as AgGrid } from '@vegaprotocol/ui-toolkit';
+import { AgGridColumn } from 'ag-grid-react';
+import type { AgGridReact } from 'ag-grid-react';
+import type { Accounts_party_accounts } from './__generated__/Accounts';
+import { getId as getRowNodeId } from './accounts-data-provider';
+
+interface AccountsTableProps {
+  data: Accounts_party_accounts[] | null;
+}
+
+interface AccountsTableValueFormatterParams extends ValueFormatterParams {
+  data: Accounts_party_accounts;
+}
+
+const getGroupId = (
+  data: Accounts_party_accounts & { __summaryRow?: boolean },
+  columnApi: ColumnApi
+) => {
+  if (data.__summaryRow) {
+    return null;
+  }
+  const sortColumnId = columnApi.getColumnState().find((c) => c.sort)?.colId;
+  switch (sortColumnId) {
+    case 'asset.symbol':
+      return data.asset.symbol;
```

Comment:
We should probably use asset.id for this. Its extremely unlikely but in theory I think we could end up with duplicate asset symbols